domainobjs/ts/__tests__/verifyingKey.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as path from "path";
+import * as fs from "fs";
+import { VerifyingKey } from "../";
+import { expect } from "chai";
+
+describe("verifyingKey", () => {
+    it("Should convert a JSON file from snarkjs to a VerifyingKey", async () => {
+        const file = path.join(__dirname, "./artifacts/test_vk.json");
+        const j = fs.readFileSync(file).toString();
+        const d = JSON.parse(j);
+        const vk = VerifyingKey.fromJSON(j);
```

Comment:
Interesting that these seems to be the only place we use `VerifyingKey.fromJSON`... I wonder why it's needed 🤔

Reply:
I suppose many classes have a `fromJSON` method for deserialization